src/praisonai-agents/praisonaiagents/runtime/builtin_rules.py

```
@@ -0,0 +1,100 @@
+            
+            # Ensure models structure exists
+            if "models" not in result:
+                result["models"] = {}
+            if "default" not in result["models"]:
+                result["models"]["default"] = {}
+            
+            result["models"]["default"]["runtime"] = runtime_value
+        
+        # Migrate cli_backend in roles
+        if "roles" in result and isinstance(result["roles"], dict):
+            for role_name, role_config in result["roles"].items():
```

Ruff (0.15.18)

[warning] Loop control variable `role_name` not used within loop body

Rename unused `role_name` to `_role_name`

(B007)